tests/SprykerEcoTest/Service/ComputopApi/ComputopApiBlowfishApiTest.php

```diff
@@ -54,7 +54,7 @@ public function testBlowfishCrypt()
         $encryptedValue = $service->getBlowfishEncryptedValue(
             self::PLAINTEXT_VALUE,
             self::LENGTH_VALUE,
-            self::PASSWORD_VALUE
+            self::PASSWORD_VALUE,
```

Comment:
```suggestion
            static::PASSWORD_VALUE,
```
Please check other places below

Reply:
Fixed